src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -1065,5 +1072,37 @@ private static DateTime ConvertToUTC(DateTime dateTime, TimeZoneInfo fromTimeZon
             var resultDateTime = new DateTimeOffset(DateTime.SpecifyKind(dateTime, DateTimeKind.Unspecified), fromTimeZone.GetUtcOffset(dateTime));
             return resultDateTime.UtcDateTime;
         }
+
+        internal static bool TryGetInt(IRContext irContext, FormulaValue value, out int outputValue)
```

Comment:
>IRContext irContext, [](http://example.com/codeflow?start=39&length=20)

We should not need this parameter.

Reply:
we need it for decimal convert to number. should i use default ircontext instead?